buildscripts/manylinux/docker_run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xe
+# Use this to make the llvmdev packages that are manylinux2010 compatible
+srcdir=$( cd "$(dirname $0)/../.."  && pwd )
+echo "srcdir=$srcdir"
+
+echo "MINICONDA_FILE=$MINICONDA_FILE"
+# Ensure the latest docker image
+image="quay.io/pypa/${MANYLINUX_IMAGE}:latest"
```

Comment:
variables in shell scripts should be uppercase by convention, so in this case I would recommend: `$IMAGE`

Reply:
or maybe even `$IMAGE_URI`?